src/frontend/komponenter/ExpansionCard.tsx

```diff
@@ -23,7 +24,9 @@ const ExpansionCard: React.FC<{ tittel: string; maxWidth: number; children: Reac
             defaultOpen
         >
             <AkselExpansionCard.Header>
-                <AkselExpansionCard.Title size="small">{tittel}</AkselExpansionCard.Title>
+                <AkselExpansionCard.Title size={tittelSize as 'small' | 'medium' | 'large'}>
```

Comment:
Her overskrives tittelSize, over den som er definiert på rad 14. 
For meg virker det som at `AkselExpansionCard.Title` ikke støtter `xsmall`, men `ExpansionCard::tittelSize` gjør det mulig å sende inn `xsmall` ?

Reply:
Er det viktig at denne tittelen er xsmall? Syns det er litt unødvendig å legge til tilpassninger som dette. Virker ikke helt som at det er meningen at aksel skal brukes på den måten🤓

Reply:
Er et ønske fra designerene at både tittelen på "Aktivitet" og "Målgruppe" blir mindre enn de er idag (for å ta mindre plass på skjermen). Som @blommish sier så støtter ikke `AkselExpansionCard.Title` størrelsen `xsmall`implisitt, men det er allikevel mulig å sende inn `xsmall` på en "hacky" måte slik som vist over (skriften blir faktisk mindre). Kanskje en bedre fremgangsmåte ville vært å sette `ExpansionCard::tittelSize` til `xsmall` slik Johan foreslår

Reply:
Bruker `size="small` etter følgende diskusjon på Slack: https://nav-it.slack.com/archives/C7NE7A8UF/p1730802744459709